specification/dimensions/pricingunit.md

```diff
@@ -0,0 +1,33 @@
+# Pricing Unit
+
+The Pricing Unit is the provider-specified measurement unit for determining the unit prices associated with the use or purchase of a specific resource and/or service. The Pricing Unit indicates how a provider rates the measured usage and purchase quantities after pricing rules, such as block pricing, are applied. Common examples include the number of hours for compute appliance runtime, gigabyte-hours for a storage appliance, or an accumulated count of requests for a network appliance or API service. In some cases, providers may opt for block pricing, such as using a million requests as the Pricing Unit instead of pricing individual requests. The Pricing Unit complements and describes how the measurement of [Quantity In Pricing Unit](#quantityinpricingunit) is conducted. The Pricing Unit pertains to pricing and cost determination and should not be confused with the [Usage Unit](#usageunit), which tracks resource and service consumption, typically at a finer granularity than the Pricing Unit.
```

Comment:
Looking to see if there's a way to make the description more concise. The second sentence seems unnecessary since we describe the block pricing provision a little further down anyway. 

```suggestion
The Pricing Unit is the provider-specified measurement unit for determining the unit prices associated with the use or purchase of a specific resource and/or service. Common examples include the number of hours for compute appliance runtime, gigabyte-hours for a storage appliance, or an accumulated count of requests for a network appliance or API service. In some cases, providers may use block pricing, like a million requests as the Pricing Unit. Pricing Unit complements the [Quantity In Pricing Unit](#quantityinpricingunit) measurement. Distinct from the [Usage Unit](#usageunit), it focuses on pricing, not resource and service consumption, often at a coarser granularity.
```

Reply:
Addressed in [a subsequent conversation](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/179#discussion_r1382309783)